tech.json

```diff
@@ -729,15 +729,17 @@
               ]
             },
             {
-              "name": "canDamageBoost",
+              "name": "canNeutralDamageBoost",
               "requires": [ "canUseEnemies" ],
-              "note": "Using the knockback from taking enemy damage to assist Samus in her movement.",
-              "devNote": "FIXME This could use a better description",
+              "note": "Using the small vertical knockback from taking enemy damage, while not holding any directional inputs, to assist Samus in her movement.",
```

Comment:
While morphed you can hold directions.  `canDamageBoost` was a better name for this in that sense.

Reply:
I thought `canNeutralDamageBoost` was better because I was worried that people would think they already know what a _damage boost_ is, and toggle it based on that.

Reply:
Sounds good, I'm cool with leaving it `canNeutralDamageBoost`